pkg/secretlock/local/passphrase_util.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package local
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/google/tink/go/subtle/random"
+)
+
+// CreateMasterKey will create a new masterkey and returns the encrypted content to be used by Lock
+// The user will then have to store it either in an env variable or a local path prior to using Lock
+// by calling either InitEnv() or InitPath() functions and passing the resulting Reader to New()
+// Calling this method and storing the returned key as mentioned above is a prerequisite for to use
+// the SecretLock service in the Aries agent.
+func CreateMasterKey(passphrase string) (string, error) {
+	masterKey := generateMasterKey(passphrase)
+
+	aeadCipher, err := createAESCipher(masterKey)
+	if err != nil {
+		return "", err
+	}
+
+	nonce := random.GetRandomBytes(uint32(aeadCipher.NonceSize()))
+	ct := aeadCipher.Seal(nil, nonce, masterKey, []byte(passphrase))
```

Comment:
why is passphrase being used as additional data?
it's supposed to be encrypting the master key.

Reply:
`masterKey` is being used to encrypt itself with `passphrase` as aad.

for decryption, we regenerate `masterKey` using `passphrase` and try to decrypt the the cipher text with `passphrase` as aad, if successful, it means `masterKey` is valid.

Reply:
this file and its logic has been removed.